### ISSUE TITLE
Accessibility: left menu and toggle-collapse aria attributes

### ISSUE THREAD

**Issue by GuySartorelli** [silverstripe/silverstripe-admin#1956]
[WCAG 2.2 A F68](https://www.w3.org/TR/WCAG20-TECHS/F68.html) 4.1.2
The [`aria-expanded` attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-expanded) is supposed to be "applied to [a] focusable, interactive control that toggles the visibility of [an] object".

This attribute is currently applied to the `div#cms-menu` element, which is the "object" in the above quote, i.e. it's the element that gets hidden or displayed.

The attribute should instead be applied to the element that controls the visibility, which is `a.toggle-collapse` and `a.toggle-expand`.
That element should also be a `<button>` rather than a `<a>` tag, should have an `aria-controls` attribute, and should be a single element instead of two elements that swap out.

## Acceptance Criteria

- [x] `div#cms-menu` element does not have `aria-expanded` attribute
- [x]  `a.toggle-collapse` and `a.toggle-expand` elements that controls the visibility of `div#cms-menu`:
    - [x] are swapped out for a single `<button>` element
    - [x] have the `aria-controls` attribute added, pointing to `cms-menu`
    - [x] have the `aria-expanded` attribute added
- [x] The changelog clearly calls out any changes to markup selectors so people can adjust their custom css/js as necessary

## Kitchen sink run
- https://github.com/creative-commoners/recipe-kitchen-sink/actions/runs/16929127593

## PRs
- https://github.com/silverstripe/silverstripe-admin/pull/1978
- https://github.com/silverstripe/silverstripe-cms/pull/3102
- https://github.com/silverstripe/developer-docs/pull/793

**Comment by GuySartorelli** [comment]
PRs merged
